pki.spec

```diff
@@ -657,8 +666,8 @@ Requires:         mvn(org.jboss.resteasy:resteasy-servlet-initializer)
 Provides:         bundled(resteasy-servlet-initializer)
 %endif
 
-%if 0%{?rhel} && 0%{?rhel} >= 10
-Requires:         tomcat9 >= 1:9.0.62
+%if 0%{?rhel} && 0%{?rhel} >= 10 && 0%{?fedora} && 0%{?fedora} >= 43
+Requires:         tomcat >= 1:10.1.33
 %else
 Requires:         tomcat >= 1:9.0.62
 %endif
```

Comment:
Sorry, since Fedora 43 build is failing due to an unrelated issue we need to be able to build and test the code on Fedora 42 using the Tomcat 10 in `@pki/11.8` repo, so this whole `%if` statement actually needs to be simplified into:
```
Requires: tomcat >= 1:10.1.33
```